Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -619,18 +619,18 @@ extension LLBuildManifestBuilder {
             name: cmdName,
             inputs: inputs + [Node.file(target.sourcesFileListPath)],
             outputs: cmdOutputs,
-            executable: self.buildParameters.toolchain.swiftCompilerPath,
```

Comment:
JW: what are "self"'s build parameters in the context of a build? Some set of default target agnostic params? something else?

Reply:
Yeah, they would be the global build parameters.  In order to support libraries, we need to have target specific build parameters as they need to be compiled with different flags based upon whether you are statically or dynamically linking to the library.